src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -2,7 +2,12 @@ import loadable from "@loadable/component"
 import { graphql } from "react-relay"
 
 const ConsignApp = loadable(() => import("./ConsignApp"))
-const OfferDetailApp = loadable(() => import("./OfferDetailApp"))
+const OfferDetailApp = loadable(() => import("./OfferDetail/OfferDetailApp"))
+
+if (typeof window !== "undefined") {
```

Comment:
I don't think this needs to be there. I was throwing darts when trying to figure out why my local environment was being mean to me. I'll try removing & see what happens.

Reply:
Yah the reason this was added to artwork page was that in certain critical path contexts we want things to be ready to go, but not strictly necessary; the `OfferDetailApp.preload()` should be sufficient

Reply:
Removed in latest commit ✅